### PR TITLE
data: Harsha1varthan

### PR DESCRIPTION
**Pull request by Harsha1varthan**
This is my first OpenSource Contribution

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4255"></a>

### CODE REVIEW

File: data/Harsha1varthan.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  "links": [
+    {
+      "name": "On Twitter",
+      "url": "https://twitter.com/harshav67037280",
+      "icon": "FaTwitter"
+    },
+     {
+      "name": "On GitHub",
+      "url": "https://github.com/Harsha1varthan",
+      "icon": "FaGitHub"
```
Comment (ChinmayMhatre) [inline data/Harsha1varthan.json:15]:
```suggestion
      "icon": "FaGithub"
```
Please update this

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]
Looks Awesome! A minor change in the file. You can also add tags with the technologies you use. They can help people discover you better

File: data/Harsha1varthan.json (hunk 1 above)
Comment (Harsha1varthan) [reply data/Harsha1varthan.json:15]: Thank you Bhai

Comment (Harsha1varthan) [reply data/Harsha1varthan.json:15]: When will I get Merged with the main repo?

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Comment by kumarsonsoff3** [comment]
Closing this one being a duplicate of 4260.  
The latest PR with added changes is merged already!

**Pull request closed without merging**